tests/blackbox/readme.md

```diff
@@ -22,15 +26,15 @@ The test suite will spin up a fresh copy of the Directus API from the current bu
 Directus, set the `TEST_LOCAL` flag:
 
 ```
-TEST_DB=cockroachdb TEST_LOCAL=true npm run test:blackbox
+TEST_DB=cockroachdb TEST_LOCAL=true pnpm -w run test:blackbox
 ```
 
 This will use `127.0.0.1:8055` as the URL for every test. Note: make sure to connect your local Directus database
 instance to the test database container found in docker-compose in this folder.
 
-### Watching for (test) changes
+### Watching for test changes
 
-Use `npm run test:blackbox:watch` to enable Jest's `--watch` mode, especially useful in combination with the flags
+Use `pnpm run test:watch` to enable Jest's `--watch` mode, especially useful in combination with the flags
 above.
 
 This _does not_ watch changes to Directus; it only watches changes to the tests.
```

Comment:
Let's remove this block, the `watch` option has been removed.

Reply:
> Should we also remove the script from `package.json`?

As long as it's consistent everywhere 👍🏻  I think we still have some `test:watch` commands, so lets make sure we clean those up together with this. It's easy enough to do `pnpm test --watch` seeing that `pnpm` passes the flags along to the underlying command 🙂

Reply:
Yes, please do! 👍

Reply:
> As we're already touching those lines (😄), should we remove the test:coverage scripts (and @vitest/coverage-c8 packages?) as well since we're not making use of it yet?

Lets remove the pnpm shorthands, but leave the dev-dep. I actively use it during development to see if I'm "done" by confirming my logic has 100% coverage 👍🏻

Reply:
Done 👍 Ready from my side 😃

Reply:
Should be possible with those commands:
```bash
pnpm -F api run test --coverage <specificTest>
pnpm -F api test -- --coverage <specificTest>
```
Would you still prefer to reintroduce the script?

Reply:
What I tend to do is `cd` into the `api` folder, and then run `p run test --coverage`. No -F required if you're already in the correct folder 🙂